spyder/api/widgets.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright © Spyder Project Contributors
+# Licensed under the terms of the MIT License
+# (see spyder/__init__.py for details)
+
+"""
+spyder.api.widgets
+==================
+
+This package exposes shareable widgets that can be used in
+third-party plugins.
+"""
+
+from spyder.widgets.tabs import Tabs
+from spyder.widgets.comboboxes import PythonModulesComboBox
+from spyder.widgets.variableexplorer.texteditor import TextEditor
```

Comment:
hmm yes, and the name could be confusing with the Editor.

maybe something straightforward  like this:
`from spyder.widgets.variableexplorer.texteditor import TextEditor as VariableExplorerTextEditor`

or create an `api/variableexporer.py` file that will expose an api of the VariableExplorer plugin like the Editor plugin?